rl4co/envs/routing/mtvrp/generator.py

```diff
@@ -256,7 +256,8 @@ def _default_open(td, remove):
     @staticmethod
     def _default_time_window(td, remove):
         default_tw = torch.zeros_like(td["time_windows"])
-        default_tw[..., 1] = float("inf")
+        #default_tw[..., 1] = float("inf")
+        default_tw[..., 1] = 4.6 # max tw
```

Comment:
Doesn't this influence the solution? If default time window is 4.6, the problem should not be a CVRP but a "relaxed" VRPTW.

The reason why I thought having "inf" is because it can generalize to any scale - for the embedding, this can be set as:
```python
time_windows = torch.nan_to_num(td["time_windows"][..., 1:, :], posinf=0.0)
```
So it shouldn't influence the calculation as describe in Section 4.1 (Attribute composition) in your paper.
What do you think?

Reply:
OK, that makes sense!